addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -566,11 +537,7 @@ odoo.define('website.tour.form_editor', function (require) {
         ...addCustomField("char", "text", '""', false),
         ...addCustomField("char", "text", "``", false),
         ...addCustomField("char", "text", "\\", false),
-        {
-            content: 'Save the page',
-            trigger: 'button[data-action=save]',
-            run: 'click',
-        },
+        ...wTourUtils.clickOnSave(),
```

Comment:
Same here, next step you can remove ` trigger: 'body:not(.editor_enable)',` to be correct